src/containers/BooksForm.js

```diff
@@ -0,0 +1,22 @@
+const BooksForm = () => {
+  // eslint-disable-next-line no-unused-vars
+  const category = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  return (
+    <>
+      <h2>Add Book here</h2>
+      <form>
+        <label for="title">Book title:</label>
```

Comment:
- [ ] `for` can't be used with React project, and most probably will be ignored. Kindly, use `htmlFor` instead

![Screenshot from 2020-12-29 12-15-39](https://user-images.githubusercontent.com/4246637/103276581-9910b900-49cf-11eb-970e-d06a67c39eb5.png)

Kindly check [this](https://reactjs.org/docs/dom-elements.html#htmlfor) out